SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -119,15 +119,15 @@ class SPATIALGDK_API USpatialClassInfoManager : public UObject
 	UPROPERTY()
 	USchemaDatabase* SchemaDatabase;
 
+	void QuitGame();
```

Comment:
Make a ticket to refactor this or refactor it into a utils file. It doesn't make sense to #include "SpatialClassInfoManager" when you want to quit the game.